### ISSUE TITLE
Inconsistent currency display in PDP

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2892]
### Expected behavior
When you are an admin, all values in the PDP should be in the shop currency.

### Actual Behavior
Some are in the shop currency, some are in the user currency.

### Steps to Reproduce the Behavior
1. Login as an admin
1. Set your user currency to be something besides the shop currency
1. Edit a product in the PDP
1. Observe that some values are reflected in the shop currency, some in the users

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.2-ce
```

**Comment by aaronjudd** [comment]
To sum it up.. the values in the PDP should reflect the chosen user language, but the language in the control/admin/sidebar should be in the base language.

**Comment by zenweasel** [comment]
@aaronjudd Do you mean currency? And yes some values in the PDP are in shop currency and some are in the user's currency so I think it's a big.

**Comment by aaronjudd** [comment]
@zenweasel yes, sorry currency

**Comment by aaronjudd** [comment]
Please screenshot and highlight what you think is a bug, I'm not sure we're in agreement.

**Comment by zenweasel** [comment]
@aaronjudd This is the only instance I am currently seeing.
![basic_reaction_product](https://user-images.githubusercontent.com/72819/30888039-e91535d0-a350-11e7-9272-c6f7a18269ff.png)

**Comment by aaronjudd** [comment]
I'm assuming the shop base value is USD?  I think we'd expect that value you are pointing at (MSRP) to be USD if the shop base value is (and I suspect price is USD, right), while the PDP should display the Egyptian currency.  What is the rest of the page looking like?

**Comment by zenweasel** [comment]
Right, shop currency is USD here and I've set user currency to be EGP.
MSRP is the only value I see that is **not** in the base currency when I edit a product. But when I create a product I see both price and MSRP with the pound sign. Price on variants doesn't show any currency symbol and correctly shows USD for the product price. (We probably don't need currency symbols for any of these inputs?)

![ybmndfisblzrb4p9j](https://user-images.githubusercontent.com/72819/30889068-88bdaab8-a356-11e7-8461-323c43bd9ab9.png)

**Comment by zenweasel** [comment]
@aaronjudd Are we still not in agreement that this is a bug?

**Comment by zenweasel** [comment]
This is still happening (but only on the "compare at" field that I can see) so I am reclassifying as a bug

![2018-01-03_11-12-10](https://user-images.githubusercontent.com/72819/34508101-22a0ad98-f077-11e7-8db9-c6b7911780b9.png)

**Comment by Akarshit** [comment]
Should be closed via #3793
